docs/porting/overview.md

```diff
@@ -1,5 +1,82 @@
-# 🔧 Overview of porting process
+# Overview of the Porting Process
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
\ No newline at end of file
+By [Phillip Carter](https://github.com/cartermp)
+
+If you've got code running on .NET Framework you may be interested in running your code on .NET Core.  This article covers evaluating if porting to .NET Core is reasonable for you, an overview of the porting process, and a list of the tools you may use to port.
```

Comment:
This sentence "porting to .NET Core is reasonable for you" reads a bit strange to me. Maybe it would read better if it said something like "porting to .NET Core is applicable to your project(s)" or maybe if you change reasonable to feasible.